src/main/rest-api/IdGenerator.js

```diff
@@ -0,0 +1,14 @@
+export class IdGenerator {
```

Comment:
Daniel, I think it would be better to make this a function that receives the parameter that you are giving to the constructor

Reply:
Yeah i think the same as aleja, it's better if you make it a function and work directly with the arguments you receive

Reply:
i think is good idea have it like a class because in a future we can use it to add more variables to the function and will be more easy to use it

Reply:
i just replaced the class with a function, I think it wont be hard if we have to change it later to a class anyway